torchsynth/module.py

```diff
@@ -58,12 +65,16 @@ def __init__(self, synthglobals: SynthGlobals, **kwargs: Dict[str, T]):
                 p.name: p for p in self.parameter_ranges
             }
             assert len(self._parameter_ranges_dict) == len(self.parameter_ranges)
+            for parameter_range in self.parameter_ranges:
+                # Put all the parameter ranges onto the device
+                parameter_range.to(self.device)
             self.add_parameters(
                 [
                     ModuleParameter(
                         value=None,
                         parameter_name=parameter_range.name,
-                        data=torch.rand((self.synthglobals.batch_size,)),
+                        # Should we use CSPRNG here? Or doesn't matter?
+                        data=torch.rand((self.synthglobals.batch_size,), device=device),
```

Comment:
I don't think it matters? We get the deterministic sampling from randomize(1) right? which randomizes all the parameters anyways. Don't think it matters during initialization.